spec/task-spec.js

```diff
@@ -0,0 +1,53 @@
+import Task from '../src/task.js'
+import {expect} from 'chai'
+
+class SampleClass extends Task {
+
+}
+
+class ImplementedTask extends Task {
+  run() {
+    // Something
+  }
+  done() {
+    // Job done!
+  }
+}
+
+describe('Task', () => {
+
+  it('should trigger an error when instantiating a new Task', () => {
+    const fn = () => new Task()
+    expect(fn).to.throw(Error)
+    expect(fn).to.throw('Can not construct Task class')
```

Comment:
Seems redundant. The test would fail if the error message changes. What about using RegEx instead of the exact error message? Or just not checking the message at all, it's only for implementor clarification purposes.